evergreen/install_baas.sh

```diff
@@ -124,7 +124,7 @@ usage()
 }
 
 WORK_PATH=""
-BAAS_VERSION=""
+BAAS_VERSION="9cc6c137728c0f1f8e26f0a517a828a1154744f1"
```

Comment:
I think what you want to do is call `install_baas.sh` in evergreen/config.yml with `-b 9cc6c137728c0f1f8e26f0a517a828a1154744f1` rather than explicitly setting this variable here.